sdk/src/main/java/com/oursky/authgear/latte/WebView.kt

```diff
@@ -27,6 +27,12 @@ internal class WebView(context: Context, val request: WebViewRequest, webContent
         this.loadUrl(url.toString())
     }
 
+    fun dispatchSignal(signal: WebViewSignal) {
+        evaluateJavascript("""
+            window.dispatchSignal("${signal.value}");
```

Comment:
I tend not to add a global function. What about

```
document.dispatchEvent(new CustomEvent("latte:event", detail: ...))
```

Reply:
@louischan-oursky This require us to duplicate the function in android / ios native code, but I think they should always execute the same function with same logic.
Why we don't want to add a global function?

Reply:
When I first saw `window.dispatchSignal`, I thought it was a browser API, turned out it wasn't. If the snippet here is `document.dispatchEvent(new CustomEvent("latte:event", detail: ...))`, I will immediately understand it is some bridging code between the webapp and the webview. 

I think we do not need to duplicate the handling logic in the SDK. The event listener still resides in the webapp.

Reply:
Initially the function `dispatchSignal` includes some serialize / parsing code for adding extra information to the event, but I removed it later because it is not needed.
Let me update the sdks to use `document.dispatchEvent(new CustomEvent("latte:event", detail: ...))`.